clap_derive/src/utils/doc_comments.rs

```diff
@@ -83,14 +83,64 @@ fn split_paragraphs(lines: &[&str]) -> Vec<String> {
         last_line += start + len;
 
         if len != 0 {
-            Some(merge_lines(&slice[..len]))
+            Some(process_paragraph(&slice[..len]))
         } else {
             None
         }
     })
     .collect()
 }
 
+fn process_paragraph(lines: &[&str]) -> String {
+    lines
+        .iter()
+        .map(|ln| ln.trim())
+         // Ignore empty lines even if they were escaped with slash
+        .filter(|ln| ln != &"\\" && !ln.is_empty())
+        // Run actual slash escaping
+        .map(|ln| {
+            let expected_len = ln.len() + 1;
+            let mut chars = ln.chars();
+            let mut ln = String::with_capacity(expected_len);
+            while let Some(c) = chars.next() {
+                if c == '\\' {
```

Comment:
Without code fence support, we could accidentally create a hard break when we shouldn't and people can end up relying on that.